docs/releases.md

```diff
@@ -40,10 +40,10 @@ Otherwise, we typically release every two weeks:
 
 ## Kubernetes Compatibility Matrix
 
-| Argo Workflows \ Kubernetes | 1.28 | 1.29 | 1.30 |
-|-----------------------------|------|------|------|
-| **3.5**                     | `✓`  | `✓`  | `✓`  |
-| **3.4**                     | `?`  | `?`  | `?`  |
+| Argo Workflows \ Kubernetes | 1.28 | 1.29 | 1.30 | 1.31 |
+|-----------------------------|------|------|------|------|
+| **3.5**                     | `✓`  | `✓`  | `✓`  | `✓`  |
+| **3.4**                     | `?`  | `?`  | `?`  | `?`  |
```

Comment:
This is still not quite correct per https://github.com/argoproj/argo-workflows/pull/13381#discussion_r1696170777

I'm thinking we just move `main` here into the table instead

Reply:
okay great, I added a `main` row